main.py

```diff
@@ -10,18 +10,25 @@
     NoviActionSendReport,
     NoviActionSetBfdData,
 )
+from napps.amlight.noviflow.pyof.v0x04.action import (
+    NoviActionAddIntMetadata as OFNoviActionAddIntMetadata,
+)
+from napps.amlight.noviflow.pyof.v0x04.action import (
+    NoviActionPopInt as OFNoviActionPopInt,
+)
+from napps.amlight.noviflow.pyof.v0x04.action import (
+    NoviActionPushInt as OFNoviActionPushInt,
+)
+from napps.amlight.noviflow.pyof.v0x04.action import (
+    NoviActionSendReport as OFNoviActionSendReport,
+)
+from napps.amlight.noviflow.pyof.v0x04.action import (
+    NoviActionSetBfdData as OFNoviActionSetBfdData,
+)
```

Comment:
```suggestion
    NoviActionPopInt as OFNoviActionPopInt,
    NoviActionPushInt as OFNoviActionPushInt,
    NoviActionSendReport as OFNoviActionSendReport,
    NoviActionSetBfdData as OFNoviActionSetBfdData,
)
```

Reply:
@italovalcy `black` and `isort` when there is an `as` it was resulting in a conflict in the linter, it automatically split it this way, the prior group of the imports it didn't complain because it didn't have any `as` or a larger statement. 

```
diff --git a/main.py b/main.py
index 33b7d47..e634ae1 100644
--- a/main.py
+++ b/main.py
@@ -12,17 +12,9 @@ from napps.amlight.noviflow.of_core.v0x04.action import (
 )
 from napps.amlight.noviflow.pyof.v0x04.action import (
     NoviActionAddIntMetadata as OFNoviActionAddIntMetadata,
-)
-from napps.amlight.noviflow.pyof.v0x04.action import (
     NoviActionPopInt as OFNoviActionPopInt,
-)
-from napps.amlight.noviflow.pyof.v0x04.action import (
     NoviActionPushInt as OFNoviActionPushInt,
-)
-from napps.amlight.noviflow.pyof.v0x04.action import (
     NoviActionSendReport as OFNoviActionSendReport,
-)
-from napps.amlight.noviflow.pyof.v0x04.action import (
     NoviActionSetBfdData as OFNoviActionSetBfdData,
 )


❯ python3 setup.py lint
running lint
Yala is running. It may take several seconds...
INFO: Finished pycodestyle
INFO: Finished isort
INFO: Finished black
INFO: Finished pylint

main.py|None:None|Imports are incorrectly sorted and/or formatted. [isort]

```

With that said, I'll leave as it is. Thanks.